spec/unit/enforcement_spec.rb

```diff
@@ -19,10 +33,20 @@
       solr_parameters[:fq].first
     end
 
+    # rubocop:disable RSpec/MessageExpectation
+    describe 'logger' do
```

Comment:
is this testing the logger method, or is it testing that `apply_gated_discovery` is calling the logger?